instagram-page-react/src/pages/profilePage/molecules/profileTabbar/ProfileTabbar.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+// Lodash
+import _map from 'lodash/map';
+
+// Styles
+import styles from './profileTabbar.module.css'
+
+// Constants
+import {profileTabbarTabs} from "./constants/profileTabbar.tabs";
+
+// Components
+import ProfileTabbarTab from "./atoms/profileTabbarTab";
+
+const renderProfileTabbar = (tab) => {
+    const{
+        ID,
+        ImageSrc,
+        Text
+    } = tab;
+
+    return <ProfileTabbarTab
+        imageSrc={ImageSrc}
+        Text={Text}
+        isSelected={ID === "POSTS" ? 1 : 0}
```

Comment:
Maintain a state `selectedTabId` and consume it like:
`isSelected={id === selectedTabId}`